src/evo/simplifiedmns.cpp

```diff
@@ -68,6 +70,10 @@ void CSimplifiedMNListEntry::ToJson(UniValue& obj, bool extended) const
     obj.pushKV("votingAddress", EncodeDestination(keyIDVoting));
     obj.pushKV("isValid", isValid);
     obj.pushKV("nVersion", nVersion);
+    obj.pushKV("nType", nType);
+    if (nType == CDeterministicMN::TYPE_HIGH_PERFORMANCE_MASTERNODE) {
+        obj.pushKV("platformHTTPPort", platformHTTPPort);
```

Comment:
why only platformhttp port?

Reply:
@PastaPastaPasta As discussed with Platform team, only `platformHTTPPort` is included in SML entry.

Reply:
Do we have a justification for this in a document somewhere?

Reply:
Yes I will send it to you.